zqd/space/space.go

```diff
@@ -37,11 +36,9 @@ var (
 	ErrSpaceNotExist       = zqe.E(zqe.NotFound, "space does not exist")
 )
 
-type Manager struct {
-	rootPath string
-	mapLock  sync.Mutex
-	spaces   map[string]*Space
-	logger   *zap.Logger
+func newSpaceID() api.SpaceID {
+	id := ksuid.New()
+	return api.SpaceID(fmt.Sprintf("sp%s", id.String()))
```

Comment:
Would you please add back the "_", so the prefix is "sp_" ?

Reply:
Done.